sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpExclusionsFileFilter.java

```diff
@@ -49,6 +58,115 @@ public boolean accept(InputFile inputFile) {
       return false;
     }
 
+    if (new AverageLineLengthCalculator(inputFile).getAverageLineLength() > DEFAULT_AVERAGE_LINE_LENGTH_THRESHOLD) {
+      LOG.debug("File [" + inputFile.uri() + "] is excluded because its default average line length is greater than " + DEFAULT_AVERAGE_LINE_LENGTH_THRESHOLD);
```

Comment:
I think we don't need word "default" here. And IMO it's better to have something more high-level in message. E.g. `.. is excluded because it's considered generated (average line length is too big)`.